test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -54,6 +54,22 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 	}
 
 	async publish( { visit = false } = {} ) {
+		// The `wpcom-editor-welcome-tour-frame` popup could get in the way
+		// of the actual publish click, so we need to check if it's visible
+		// and click `Skip` to close it.
+		const isWelcomeDialogDisplayed = await driverHelper.isElementLocated(
```

Comment:
The `Nux Welcome Tour Modal` ([source here](https://github.com/Automattic/wp-calypso/tree/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux)), started appearing all of the sudden on some test sites, and was blocking the `publish` click. Does anyone know why it'd start appearing? It seemed to be related to the new Gutenberg version 🤔  Or maybe it was already there and just changed its coordinates? @razvanpapadopol do you happen to know anything about this? 🙇🏻

Reply:
Thanks for the ping, @fullofcaffeine !
There was a recent change only for mobile viewports. Me and @chriskmnds swapped the welcome modal ([source](https://github.com/Automattic/wp-calypso/tree/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-modal)) with the new welcome tour ([source](https://github.com/Automattic/wp-calypso/tree/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour)) that was already being used for desktop resolutions. So at the beginning of this week (Aug 30) the mobile NUX went from [this broken state](https://github.com/Automattic/wp-calypso/issues/52280#issuecomment-906195184) to [this one](https://github.com/Automattic/wp-calypso/issues/52280#issuecomment-908172746).